BitcoinApp/TransactionViewController/TransactionCell.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  TransactionCell.swift
+//  BitcoinApp
+//
+//  Created by Zhassulan Aimukhambetov on 12/6/19.
+//  Copyright © 2019 Zhassulan Aimukhambetov. All rights reserved.
+//
+
+import UIKit
+
+class TransactionCell: UITableViewCell {
+
+    @IBOutlet weak var numberLabel: UILabel!
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var amountLabel: UILabel!
+    @IBOutlet weak var typeLabel: UILabel!
+    
+    private let cellOddColor = UIColor(red: 10/255, green: 10/255, blue: 10/255, alpha: 1)
+    private let cellEvenColor = UIColor(red: 5/255, green: 5/255, blue: 5/255, alpha: 1)
+    private let cellSelectedColor = UIColor(red: 25/255, green: 25/255, blue: 25/255, alpha: 1)
+    
+    func configure(_ transaction: Transaction, indexPath: IndexPath) {
+        let backgroundView = UIView()
+        backgroundView.backgroundColor = cellSelectedColor
+        self.backgroundColor = indexPath.row % 2 == 0 ? cellOddColor : cellEvenColor
```

Comment:
The Cell itself should never know is it an odd or even in the tableView. 
Add the argument `isOdd` to your `configure` method and use it instead.